doc/releases/changelog-dev.md

```diff
@@ -214,6 +214,13 @@
   of the computed qubit operator, if imaginary components are smaller than a threshold. 
   [(#4639)](https://github.com/PennyLaneAI/pennylane/pull/4639)
 
+* Updated `qml.device`, `devices.preprocessing` and the `tape_expand.set_decomposition` context 
+  manager to bring `DefaultQubit2` to feature parity with `default.qubit.legacy` with regards to 
+  using custom decompositions. The `DefaultQubit2` device can now be included in a `set_decomposition` 
```

Comment:
```suggestion
  using custom decompositions. The `DefaultQubit` device can now be included in a `set_decomposition` 
```

Reply:
we'll probably need a nice big cleanup for the changelog in this respect, but yep

Reply:
I thought we were leaving it `DefaultQubit2` everywhere for lack of ambiguity and letting Tom decide a uniform way of describing things when he organizes the changelog?